src/pages/Home/Home/index.jsx

```diff
@@ -1,7 +1,23 @@
 import React from "react";
-
+import { useDispatch, useSelector } from "react-redux";
+import { createButton } from "../../../store/actions";
+import TaskBoard from "../TaskBoard";
+import style from "./index.module.scss";
 const Home = () => {
-  return <div>Home</div>;
+  const dispatch = useDispatch();
+  const createButtonState = useSelector((state) => state.createButton);
+  return (
+    <div className={style.container}>
+      <h1>Add Task</h1>
+      <button
+        disabled={createButtonState}
+        onClick={() => dispatch(createButton())}
```

Comment:
can we add another variable for this function.
```suggestion
        onClick={dispatchCreateButton}
```